core/modules/wdb/QueryRunner.cc

```diff
@@ -344,7 +359,7 @@ void QueryRunner::_transmit(bool last, unsigned int rowCount, size_t tSize) {
 
 
 void QueryRunner::_sendBuf(xrdsvc::StreamBuffer::Ptr& streamBuf, bool last,
-                           util::TimerHistogram& histo, std::string const& note) {
+                           util::TimerHistogram& histo, string const& note) {
```

Comment:
In fact, inconsistent use of "std::" seems to pervade this update across a wide swath of classes!

Reply:
'using namespace std' in .cc files is removes a lot of unnecessary characters making the code easier to read. It makes sense to remove 'std::' on definitions everyone is familiar with, but there's no reason not to leave it on less commonly used functions. It's a change @iagaponenko and I have been doing as we come across them. The change adds a lot of chaff to pull requests.